lib/IRGen/IRGenModule.cpp

```diff
@@ -2438,7 +2438,7 @@ bool swift::writeEmptyOutputFilesFor(
     auto *clangImporter = static_cast<ClangImporter *>(
       Context.getClangModuleLoader());
     llvmModule->setTargetTriple(
-      clangImporter->getTargetInfo().getTargetOpts().Triple);
+        llvm::Triple(clangImporter->getTargetInfo().getTargetOpts().Triple));
```

Comment:
What was `getTargetOpts().Triple` changed to 🤔?

Reply:
It is a `std::string` now I believe.

Reply:
It’s `llvm:Module::setTargetTriple` that switched to accepting a triple.